### PR TITLE
Add Mobile Wallets to core software downloads

### LINKED ISSUE #689: add mobile wallets to download section
https://play.google.com/store/apps/details?id=com.decred.dcrandroid.mainnet

https://apps.apple.com/us/app/decred-wallet/id1462247643

### PR DESCRIPTION
**Pull request by Jujhar**
Screen Shot 2019-07-19 at 10 03 59 PM

Fixes #689 

- Same icon as decredtion card.
- No release notes link as there are seperate repos for andriod and ios and thought it would be too cluttered if I put both.

### CODE REVIEW

File: src/data/downloads/links.yml [hunk 1]
```diff
@@ -20,6 +20,14 @@ decrediton:
     - title: Release notes
       url: https://github.com/decred/decred-binaries/blob/master/release-notes.md#decrediton-v140
 
+mobilewallets:
+    - title: Android
+      url: https://play.google.com/store/apps/details?id=com.decred.dcrandroid.mainnet
+      is_direct_download: true
```
Comment (jholdstock) [inline src/data/downloads/links.yml]: These are not direct download links, they are links to the play/app store

Comment (Jujhar) [reply src/data/downloads/links.yml]: I know but arrow points to the right then is that okay?

Comment (jholdstock) [reply src/data/downloads/links.yml]: I think so yes, lets keep it consistent with the rest of the page.

**Comment by morphymore** [comment]
Hello @jholdstock & @Jujhar, here's an icon for Mobile Wallets.

[ic_mobile.svg.zip](https://github.com/decred/dcrweb/files/3420265/ic_mobile.svg.zip)

Preview:
ic_mobile

**Comment by Jujhar** [comment]
Added the icon

**Pull request merged**